src/main/kotlin/no/nav/familie/tilbake/service/dokumentbestilling/felles/pdf/DokprodTilHtml.kt

```diff
@@ -1,8 +1,12 @@
 package no.nav.familie.tilbake.service.dokumentbestilling.felles.pdf
 
+import java.net.URLEncoder
+import java.nio.charset.StandardCharsets
+
 object DokprodTilHtml {
 
-    fun dokprodInnholdTilHtml(dokprod: String): String {
+    fun dokprodInnholdTilHtml(tekst: String): String {
```

Comment:
er det mulig å refaktuere denne metoden? Cognitive Complexity er ganske høy (38)

Reply:
Veldig vanskelig å bryte opp siden det er så mange avhengigheter til tidliger linjer innad i hvert avsnitt. Hvis du synes det er nødvendig, så bør vi opprette et eget favro-kort for det.